### PR TITLE
feat: Collection notification - drops minting is live

### LINKED ISSUE #9452: Collection notif - drops minting is live
image

## Design
- in small things handoff: https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=1287-18959&mode=design&t=wdrvHvIUq89tvpZ0-11
- please ignore the rest of landing page - the important part is the pill shape notification 
- same notification as with migration ready 

notification will be visible when the drop is minting - after the drop ends, the notif is hidden

if there is collection/echo it will redirect to the underlying drop /drops/echo etc...

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Feature

  ## Needs Design check

  - @exezbcz please review

  ## Context

  - [x] Closes #9452

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [x] My fix has changed UI

/ahp/collection/95

image

/ahp/collection/91
image

  ## Copilot Summary
  copilot:summary

  copilot:poem

### CODE REVIEW

**Comment by exezbcz** [comment]
thanks!

**Review by preschian — APPROVED** [review]

File: components/collection/CollectionHeader/CollectionBanner.vue [hunk 1]
```diff
@@ -42,6 +42,28 @@
...
+            </div>
+            <div class="w-4 h-[1px] bg-separator-line-color mx-2" />
+            <nuxt-link
+              class="flex items-center font-bold my-2"
+              :to="`/${relatedDrop.chain}/drops/${relatedDrop.alias}`">
+              {{ $t('drops.viewDrop') }}
+            </nuxt-link>
+          </div>
+        </div>
+
```
Comment (daiagi) [inline components/collection/CollectionHeader/CollectionBanner.vue:66]: better as a new component i think

**Review by daiagi — APPROVED** [review]
otherwise LGTM

**Comment by prury** [comment]
not showing on mobile: 

image

File: components/collection/CollectionHeader/CollectionBanner.vue (hunk 1 above)
Comment (Jarsen136) [reply components/collection/CollectionHeader/CollectionBanner.vue]: updated

**Comment by Jarsen136** [comment]
> not showing on mobile:
> 
> image

✅ Fixed

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
changed to `api.tx.balances.transfer` -> `api.tx.balances.transferAllowDeath`

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 7.82 USD/DOT ~ 3.836 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x6870fb3460471174d05b6560f5277cdaddccb2ca3567150fc120925e8488bf80](https://polkadot.subscan.io/extrinsic/0x6870fb3460471174d05b6560f5277cdaddccb2ca3567150fc120925e8488bf80)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**